erdgo/blockchain/proxy.go

```diff
@@ -223,7 +233,12 @@ func (ep *elrondProxy) getHTTP(endpoint string) ([]byte, error) {
 
 func (ep *elrondProxy) postHTTP(endpoint string, data []byte) ([]byte, error) {
 	url := fmt.Sprintf("%s/%s", ep.proxyURL, endpoint)
-	response, err := http.Post(url, "", strings.NewReader(string(data)))
+	request, err := http.NewRequest("POST", url, strings.NewReader(string(data)))
```

Comment:
you could have used `http.MethodPost` instead of `"POST"`

Reply:
also you could have used `bytes.NewReader(data)` instead of casting to string and creating a string reader